cmd/createcluster.go

```diff
@@ -558,27 +570,40 @@ func generateKeys(numDVs int) ([]tbls.PrivateKey, error) {
 }
 
 // createDepositDatas creates a slice of deposit datas using the provided parameters and returns it.
-func createDepositDatas(withdrawalAddresses []string, network string, secrets []tbls.PrivateKey) ([]eth2p0.DepositData, error) {
+func createDepositDatas(withdrawalAddresses []string, network string, secrets []tbls.PrivateKey, depositAmounts []eth2p0.Gwei) ([][]eth2p0.DepositData, error) {
 	if len(secrets) != len(withdrawalAddresses) {
 		return nil, errors.New("insufficient withdrawal addresses")
 	}
+	if len(depositAmounts) == 0 {
+		return nil, errors.New("deposit amounts must not be empty")
```

Comment:
nit (same as above):
```suggestion
		return nil, errors.New("empty deposit amounts")
```